curl/basic.go

```diff
@@ -21,7 +21,7 @@ func (b *Bool) Set(value string) error {
 }
 
 func (b Bool) Get() interface{} { return bool(b) }
-func (b Bool) Type() string     { return "" }
+func (b Bool) Type() string     { return "bool" }
```

Comment:
I vaguely remember having to set this to an empty string otherwise the `pflag` package would mistakenly assume the parameter to nto be a boolean flag (even when `IsBoolFlag` is called).

Reply:
This is necessary for https://github.com/segmentio/kubectl-curl/pull/10/files/7c3de1c025cc4c453aba5ea02f993e4d570dd89a#diff-4f398a0be3334bcbf9e2df90b2244fceff54de3e991936a2606f50f1c8ea6be3R108-R109 to work. Also those lines are assuming if one does `-v` or `--verbose` that it would be true and not supporting a use case like `--verbose=false` which doesn't seem practical.

Reply:
Do you have an example where this change fails something previously working?

Reply:
Could we check for `Type() == ""` instead of `Type() == "bool"` ?

Reply:
I'm not able to find the code where type was being used, so maybe it's fine to assume it'll be OK. If it seems to work I have no concerns 👍